src/Reprocessing_Internal.re

```diff
@@ -14,7 +14,8 @@ let getProgram
   Gl.compileShader ::context vertexShader;
   let compiledCorrectly =
     Gl.getShaderParameter
-      ::context shader::vertexShader paramName::Gl.Compile_status == 1;
+      ::context shader::vertexShader paramName::Gl.Compile_status
+    == 1;
```

Comment:
What version of `refmt` are you using? Could you make sure you have the latest and make the refmt-width is 100? This seems pretty wrong.

Reply:
`Reason 1.13.7 @ d13c6bd`

Looks like this was just a matter of my width being set to 80 instead of 100.

Should it be 100 then? At least for this project?

Reply:
@bsansouci it should be the default refmt width until refmt supports a project-local .refmt settings file or similar.